templates/cloudformation/aws-resources.yaml

```diff
@@ -15,6 +19,13 @@ Parameters:
     Default: 'change-me'
     Description: AEM License for Deployment
     Type: String
+  AEMKeyStorePasswordParameter:
+    Default: 'change-me'
+    Description: Java Keystore password used in AEM Author and Publish
+    Type: String
```

Comment:
Could you please confirm whether `Type: String` indicates secure string?
According to https://aws.amazon.com/about-aws/whats-new/2018/08/aws-cloudformation-introduces-dynamic-references-to-support-aws-/ , CloudFormation should already support secure string.

Reply:
Hi Cliff,

I have investigated this further and concluded that AWS only supports Secure String for Systems Manager at the moment